### PR TITLE
:art: Add koifish animation to the login/register

### PR DESCRIPTION
**Pull request by feliciofilipe**
Adds a Koi Fish animation to the login and register forms.

To note that the way it is implemented (with JQuery) it is not the desirable one but it works.
A medium to long term goal should be to rewrite this feature.

Screenshot 2021-09-03 at 21-33-45 CoderDojo Braga

### CODE REVIEW

File: components/Login/index.js [hunk 1]
```diff
@@ -15,6 +15,8 @@ function Login() {
 
   return (
     <Form name="login" onFinish={onFinish}>
+      <div id="fishesHolder" class="fishesHolder"></div>
```
Comment (nelsonmestevao) [inline components/Login/index.js:18]:
```suggestion
```

File: pages/login.js [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
 import AuthenticationLayout from "~/components/layouts/AuthenticationLayout";
 import LoginForm from "~/components/Login";
+import Script from 'next/script';
 
 function Login() {
   return (
-    <AuthenticationLayout>
-      <LoginForm />
-    </AuthenticationLayout>
+    <>
```
Comment (nelsonmestevao) [inline pages/login.js:8]:
```suggestion

```

File: pages/login.js [hunk 3]
```diff
@@ -1,12 +1,16 @@
...
+      <AuthenticationLayout>
+        <Script src="/vendor/koi-animation.js" />
+        <LoginForm />
+      </AuthenticationLayout>
+    </>
```
Comment (nelsonmestevao) [inline pages/login.js:13]:
```suggestion
```

File: pages/login.js [hunk 4]
```diff
@@ -1,12 +1,16 @@
...
+import Script from 'next/script';
 
 function Login() {
   return (
-    <AuthenticationLayout>
-      <LoginForm />
-    </AuthenticationLayout>
+    <>
+      <AuthenticationLayout>
+        <Script src="/vendor/koi-animation.js" />
```
Comment (nelsonmestevao) [inline pages/login.js:10]: Move this inside AuthenticationLayout.

**Review by nelsonmestevao — CHANGES_REQUESTED** [review]
It looks amazing and super cute! :tropical_fish: 

:stop_sign: Unfortunately, this still breaks mobile.

It's a little bit heavy, maybe we will need to reduce the number of fishs.

**Comment by nelsonmestevao** [comment]
Maybe the easiest way to fix on mobile is to  not show it :thinking:

**Comment by miguelbrandao** [comment]
I think it looks pretty cute and fun, but it almost doubles the download size (from about 250kB to 430kB).

Besides, I think there are too many fishes; a third or a quarter should be enough. This should also prevent performance issues in lower spec machines, although this is only a concern, as I have not tested it.

On another point, I agree with @nelsonmestevao  that this should be a component. Also, if we are going to merge this, I think we should be honest with ourselfs: this will probably never be rewritten and will remain jQuery for all of eternity :joy:

**Comment by nelsonmestevao** [comment]
We probably can stop this being downloaded on mobile too. At least on mobile it would be good to save some bandwidth.

**Review by nelsonmestevao — APPROVED** [review]
Really cool 🐠🐟🐡

**Pull request merged**